app/static/styles/main.css

```diff
@@ -129,4 +129,42 @@ li a:hover:not(.active) {
 
 .active {
   background-color: #30519C;
+}
```

Comment:
Since these are specific to the 404 page, maybe we could make it a separate file?